src/index.html

```diff
@@ -10,7 +10,28 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class ="search-form search-form_big" data-qa="big">
+      <label class ="search-form__label search-form__label_big">
```

Comment:
It's recommended to add `for` attribute in `label` tag for better accessibility. The `for` attribute specifies which form element a label is bound to. It should be the same value as you have on input `id`